front/src/stories/layout.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import Layout from "~/components/layout";
+
+type T = typeof Layout;
+type Story = ComponentStory<T>;
+
+export default {
+  title: "Layout",
+
+  component: Layout,
+} as ComponentMeta<T>;
+
+export const Default: Story = () => (
```

Comment:
上のexport defaultで制御すべき

```suggestion
export const Default: Story = {};
```

Reply:
[4dfddbb](https://github.com/Growthers/enshita/pull/49/commits/4dfddbb58143a7afd8f48b83aefc8e3eafe1a764)
削除しました。